src/components/common/Navbar/navbar.scss

```diff
@@ -0,0 +1,37 @@
+
+@import "../../../styles/vendors/include-media";
+
+.navbar {
+  [data-store-search-input] {
+    flex: 1 0 100%;
+    order: 2;
+  }
+  @include media(">=tablet") {
+    [data-store-search-input] {
+      max-width: 37.5rem;
+      order: 1;
+    }
+    .navlinks {
+      flex: 1 0 100%;
+      order: 3;
+    }
+    .cart-toggle { order: 2; }
```

Comment:
Nice! Flexbox 💛 

Should we use tablet or notebook there?

Reply:
I've just checked with flip about the breakpoint, I'll wait for renatinha to update it :)